app/templates/public.hbs

```diff
@@ -33,14 +33,16 @@
           {{/if}}
         {{else}}
           {{#if this.displaySideMenu}}
-            <div class="ui fluid side-menu">
+          <UiSticky class="side-menu" @offset={{10}} @context={{'#event-detail'}} @observeChanges={{true}}>
```

Comment:
changed the context to the column which was loading dynamically to solve scrolling issue.

Reply:
@mariobehling I have tested this with the previous issue raised by you and it works fine now. Please let me know if any further actions are required.